daprdocs/content/en/developing-applications/integrations/cloud-providers/authenticating-azure.md

```diff
@@ -0,0 +1,399 @@
+---
+type: docs
+title: "Authenticating to Azure"
+linkTitle: "Authenticating to Azure"
+description: "How to authenticate Azure components using Azure AD and Managed Identities"
```

Comment:
Azure Active Directory

Reply:
No in this case it should be "Azure" because it's for components authenticating with Azure. Also, for symmetry with the AWS one: https://docs.dapr.io/developing-applications/integrations/cloud-providers/